### PR TITLE
Add uniswap-v2 subgraph

### PR DESCRIPTION
**Pull request by steegecs**

### CODE REVIEW

File: subgraphs/uniswap-v2/src/common/getters.ts [hunk 1]
```diff
@@ -44,7 +42,7 @@ export function getOrCreateDex(): DexAmmProtocol {
     protocol = new DexAmmProtocol(FACTORY_ADDRESS)
     protocol.name = "Uniswap v2"
     protocol.slug = "uniswap-v2"
-    protocol.schemaVersion = "1.0.0"
+    protocol.schemaVersion = "1.0.1"
     protocol.subgraphVersion = "1.0.0"
```
Comment (this-username-is-taken) [inline subgraphs/uniswap-v2/src/common/getters.ts:46]: let's increment this as well

Comment (this-username-is-taken) [inline subgraphs/uniswap-v2/src/common/getters.ts:46]: let's increment this as well

File: subgraphs/uniswap-v2/src/common/getters.ts [hunk 2]
```diff
@@ -77,18 +75,19 @@ export function get(poolAddress: string): _LiquidityPoolAmounts {
...
-    if (!token) {
-        token = new Token(tokenAddress.toHexString())
-        token.symbol = fetchTokenSymbol(tokenAddress)
-        token.name = fetchTokenName(tokenAddress)
-        token.decimals = fetchTokenDecimals(tokenAddress)
-        token.save()
-    }
-    return token
-}
+// export function getOrCreateToken(tokenAddress: Address): Token {
```
Comment (this-username-is-taken) [inline subgraphs/uniswap-v2/src/common/getters.ts:78]: why not remove?

File: subgraphs/uniswap-v2/src/common/tokens.ts [hunk 3]
```diff
@@ -1,87 +1,120 @@
...
+      token.decimals = DEFAULT_DECIMALS
+      token.save()
+  }
+  return token
 }
 
-export function fetchTokenName(tokenAddress: Address): string {
-  let contract = ERC20.bind(tokenAddress)
-  let contractNameBytes = ERC20NameBytes.bind(tokenAddress)
+// export function fetchTokenSymbol(tokenAddress: Address): string {
```
Comment (this-username-is-taken) [inline subgraphs/uniswap-v2/src/common/tokens.ts:40]: why are these commented out? Should either uncomment or remove

Comment (this-username-is-taken) [inline subgraphs/uniswap-v2/src/common/tokens.ts:40]: why are these commented out? Should either uncomment or remove

File: subgraphs/uniswap-v2/src/common/getters.ts (hunk 1 above)
Comment (steegecs) [reply subgraphs/uniswap-v2/src/common/getters.ts:46]: I did so

Comment (steegecs) [reply subgraphs/uniswap-v2/src/common/getters.ts:46]: Done

File: subgraphs/uniswap-v2/src/common/getters.ts (hunk 2 above)
Comment (steegecs) [reply subgraphs/uniswap-v2/src/common/getters.ts:78]: Removed

File: subgraphs/uniswap-v2/src/common/tokens.ts (hunk 3 above)
Comment (steegecs) [reply subgraphs/uniswap-v2/src/common/tokens.ts:40]: Removed

Comment (steegecs) [reply subgraphs/uniswap-v2/src/common/tokens.ts:40]: Removed

**Pull request merged**